node_2026/stream_sync.sh

```
@@ -0,0 +1,159 @@
+#!/bin/bash
+# Script for live DASH/HLS streaming lossy audio as AAC and/or archiving lossless audio as FLAC
+
+# Ensure system binaries are in PATH
+export PATH=/usr/bin:/usr/local/bin:/usr/sbin:/bin:$PATH
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.11.0)

[warning] SCRIPT_DIR appears unused. Verify use (or export if used externally).

(SC2034)